source/_integrations/stookalert.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: "Stookalert"
+description: "Instructions on how to use Stookalert data within Home Assistant"
+logo: stookalert.png
+ha_category:
+  - binary-sensor
+  - environment
+ha_release: 0.103
```

Comment:
Needs to point to the release in which the integration was released.

Reply:
Thanks for your comments! 
What release would this be?